doc/source/examples/JobAnalysis/job_analysis.ipynb.rst

```diff
@@ -279,50 +285,50 @@ Another use case may be to analyze the results from one or more jobs. For this,
        .remove_uniform_fields()
    )
 
-   ja_adf.display_table()
+   ja_adf_expanded.display_table()
 
 ======= ======== ================= ============= =============== ======== =================== ===============================================
 Name    CPUTime  InputAdfBasisType InputAdfXcGga Energy [kJ/mol] AtomType Charge              AtomCoords
 ======= ======== ================= ============= =============== ======== =================== ===============================================
-adf.011 2.780308 DZ                None          -1316.30        O        -0.8416865250737331 [-2.17062120e-04 3.82347777e-01 0.00000000e+00]
```

Comment:
Why are these values different?

Reply:
CPU time expected to differ on re-running notebook